pkg/parser/common.go

```diff
@@ -271,6 +271,26 @@ func SplitDDL(stmt ast.StmtNode, schema string) (sqls []string, err error) {
 
 			v.Specs = []*ast.AlterTableSpec{spec}
 
+			// handle `alter table t1 add column (c1 int, c2 int)`
+			if spec.Tp == ast.AlterTableAddColumns && len(spec.NewColumns) > 1 {
+				columns := spec.NewColumns
+				spec.Position = &ast.ColumnPosition{
+					Tp: ast.ColumnPositionNone, // otherwise restore will become "alter table t1 add column (c1 int)"
+				}
+				for _, c := range columns {
+					spec.NewColumns = []*ast.ColumnDef{c}
+					bf.Reset()
+					err = stmt.Restore(ctx)
+					if err != nil {
+						v.Specs = specs
+						v.Table = table
+						return nil, terror.ErrRestoreASTNode.Delegate(err)
+					}
+					sqls = append(sqls, bf.String())
+				}
+				continue
```

Comment:
why need this `continue` here?

Reply:
it's the continue of `for _, spec := range specs`. because we now know this `spec` is a add column statement and restored its SQL for every column, we could skip below logic about "restore whole spec to SQL" and "handle rename table"

Reply:
you can add a comment to highlight this.